docs/kit/logistics/kit-testing/README.md

```diff
@@ -0,0 +1,104 @@
+# Kit Testing
+
+Prior to SR2025, Kit Collation and Kit Testing were separate events.
+From SR2025, we have combined the two events to reduce the number of times that volunteers need to travel for kit events over the summer.
+The key improvements that have allowed for this are the increased automation of the test scripts and the included functionality of recording board asset codes during testing.
+This allows for the removal of the manual collation step.
+
+All the automated test scripts are included in the [kit-test-scripts package](https://github.com/srobo/kit-test-scripts).
+This can be installed through pip:
+
+```bash
+pip install sr-kit-test
```

Comment:
Given it's an SR package, can you add `srobo` as an owner of the package.

Reply:
Done